### ISSUE TITLE
The host runtime fails to start

### ISSUE THREAD

**Issue by ratulb** [wasmCloud/wasmcloud-otp#241]
My environment is:

DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic

1) Downloaded and extracted https://github.com/wasmCloud/wasmcloud-otp/releases/download/v0.50.0/x86_64-linux.tar.gz 

2) Installed NATS and other requisites as per "https://wasmcloud.dev/overview/installation/".

When I run:
`bin/wasmcloud_host foreground`

- Getting following messages:

releases/0.50.0/wasmcloud_host.sh: error while loading shared libraries: libtinfo.so.6: cannot open shared object file: No such file or directory
Unusable Erlang runtime system! This is likely due to being compiled for another system than the host is running

Subsequently:

Tried building `host_core` after installing erlang  and elixir - then I face following issue:

make build
mix deps.get
** (Mix.Config.LoadError) could not load config config/dev.exs
    ** (ArgumentError) expected config file to return keyword list, got: nil
    (mix) lib/mix/config.ex:241: Mix.Config.validate!/1
    (mix) lib/mix/config.ex:168: Mix.Config.read!/1
    (mix) lib/mix/config.ex:195: anonymous fn/2 in Mix.Config.read_wildcard!/1
    (elixir) lib/enum.ex:1623: Enum."-reduce/3-lists^foldl/2-0-"/3
    (stdlib) erl_eval.erl:670: :erl_eval.do_apply/6
    (stdlib) erl_eval.erl:878: :erl_eval.expr_list/6
    (stdlib) erl_eval.erl:404: :erl_eval.expr/5
Makefile:28: recipe for target 'deps' failed
make: *** [deps] Error 1

**Comment by autodidaddict** [comment]
@ratulb sorry about the delay. I didn't get notified that this issue had been created and I suspect no one else on the team did either.

**Comment by ratulb** [comment]
I am on amd64. Thanks for looking into it.

**Comment by brooksmtownsend** [comment]
I was pretty sure that I had this running on `ubuntu:18.04`, but after a bit of investigation looks like the dynamic library isn't found for the BEAM:
```
+ ldd beam.smp
./beam.smp: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by ./beam.smp)
        linux-vdso.so.1 (0x00007ffe335f2000)
        libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007fd19b130000)
        libtinfo.so.6 => not found
        libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007fd19af13000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007fd19acf4000)
        librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007fd19aaec000)
        libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007fd19a763000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fd19a3c5000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007fd19a1ad000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fd199dbc000)
        /lib64/ld-linux-x86-64.so.2 (0x00007fd19b7ad000)
```

`libncursesw6` is not available on ubuntu 18.04 bionic. Attempting this on `20.04` works just fine, I think this is likely because we build our x86 linux binary from the github runner `ubuntu:latest`, which is `20.04`.

@ratulb the short-term fix will be to use 20.04 if you are able to, in the slightly longer term we'll need to determine if that package is necessary and downgrade our runner to 18.04 as long as it doesn't break any of the more updated versions of linux operating systems. In regards to your second issue, the highest version we've tested with is `Erlang/OTP 24`, which is almost the highest released version on https://www.erlang.org/downloads/24.1, I'd suggest not using a development version of Erlang/OTP for this, but would happily entertain a PR fixing that issue.
